docs/debugging.md

```diff
@@ -42,6 +42,10 @@ By default, Bolt tries to connect over the standard SSH port 22. If you need to
 
 Make sure you've specified the `winrm` protocol for the target. You can either include it in the name of the target (`winrm://hostname.example.com`), pass `--transport winrm` on the CLI, or set the transport in your Bolt config or inventory. By default, Bolt will try to connect over SSH.
 
+## Plan functions are not logging to the console
+
+The default log level for the console is `warn`. When a `notice` function is used in a plan, it will not be printed to the console. You can see these messages by running the plan with the `--debug` parameter or by configuring the (console log level)(https://puppet.com/docs/bolt/latest/bolt_configuration_options.html#log-file-configuration-options).  You could also use the `warning` log function instead of `notice`. Another option is to use the `out::message` plan function which will print string messages to the console, regardless of log level. 
```

Comment:
Should we be more prescriptive here?

If you have messages you always want to be printed use the `out::message` plan function instead of `notice`. If you are temporarily debugging a plan use `warning` or `out::message` to print messages. To permanently add debug messages to a plan use `notice` and pass `--debug` when running the plan.

Reply:
Yeah, being prescriptive here makes sense. I think i could also add a note https://puppet.com/docs/bolt/latest/writing_plans.html#puppet-log-functions describing what to expect between logger and outputter.

Reply:
Yeah I think rewriting that section to be primarily out::message based makes sense.


also puppetdb should probably have it's own section